python/TestHarness/TestHarness.py

```diff
@@ -794,6 +794,14 @@ def writeResults(self):
 
                 # Create empty key based on TestDir, or re-inialize with existing data so we can append to it
                 self.options.results_storage[job.getTestDir()] = self.options.results_storage.get(job.getTestDir(), {})
+
+                # If output has been stored in separate files, don't make additional copies by
```

Comment:
I assume this is one of the cases of duplication? 

This makes me realize how badly we need more TestHarness testing and documentation :(

Reply:
indeed.

```python
    def writeResults(self)
```
is under much development in my PR. We do have unittesting for the sep-file(s) arguments (I think we do anyway), just not for PBS usage of sep-files. I'll have to come up with some mocking to make this work.